app/code/Magento/InventoryLowStockNotification/Test/Integration/Block/Adminhtml/Product/Lowstock/GridTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryLowStockNotification\Test\Integration\Block\Adminhtml\Product\Lowstock;
+
+use Magento\InventoryLowStockNotification\Block\Adminhtml\Product\Lowstock\Grid;
+use Magento\InventoryLowStockNotification\Model\ResourceModel\Product\Lowstock\CollectionFactory;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @magentoDbIsolation enabled
+ * @magentoAppIsolation enabled
+ */
+class GridTest extends TestCase
```

Comment:
Unfortunately, this test is totally not applicable (set of bad practice and antipatterns)
Pls delete it and create new one

In new test, you need to work with `\Magento\InventoryLowQuantityNotification\Model\ResourceModel\LowQuantityCollection`
You will check data from collection based on preconditions